packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.test.tsx

```diff
@@ -1,15 +1,14 @@
-/* tslint:disable:no-unused-variable */
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
-/* tslint:enable:no-unused-variable */
 import * as ReactTestUtils from 'react-dom/test-utils';
 import * as renderer from 'react-test-renderer';
-import { mount } from 'enzyme';
-
-import { KeyCodes, resetIds } from '../../Utilities';
 import { Dropdown } from './Dropdown';
 import { DropdownBase } from './Dropdown.base';
-import { DropdownMenuItemType, IDropdownOption, IDropdown } from './Dropdown.types';
+import { DropdownMenuItemType, IDropdown, IDropdownOption } from './Dropdown.types';
+import { KeyCodes, resetIds } from '../../Utilities';
+import { mount } from 'enzyme';
+/* tslint:disable:no-unused-variable */
```

Comment:
Can you remove these lines, or just revert this file entirely since you didn't end up making any changes here?

Reply:
will do